src/mono/netcore/System.Private.CoreLib/README.md

```diff
@@ -0,0 +1,8 @@
+# System.Private.CoreLib for Mono Runtime
+This folder contains Mono runtime specific build of System.Private.CoreLib.dll. The actual implementation is located in two folders. The code which has hard dependency on Mono VM implementation can be found under [src](src/) and common code in [shared](/src/libraries/System.Private.CoreLib/src/README.md).
```

Comment:
> [shared]

We are not using "shared" as the directory name anymore.

Reply:
What name would you suggest?

Reply:
Use the directory like in https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/README.md ?